src/device.adb

```diff
@@ -367,6 +367,40 @@ package body Device is
 
    end Get_CPU_From_Project;
 
+   --------------------------------
+   -- Get_User_Tags_From_Project --
+   --------------------------------
+
+   procedure Get_User_Tags_From_Project
+      (Self         : in out Spec;
+       Spec_Project : Project_Type)
+   is
+      Interrupts : constant Attribute_Pkg_String :=
```

Comment:
Is the name **Interrupts** a copy and paste mistake? Seems like this and **Interrupt_List** are odd name choices for this procedure.